### PR TITLE
Fix(image): get convert and extract image from env

### PR DESCRIPTION
**Pull request by FogDong**
**What type of PR is this?**
/kind deprecation
/area serving

**What this PR does / why we need it**:
Fix(image): get convert and extract image from env

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @simon-cj @judgeeeeee 

**Does this PR introduce a user-facing change?**:

```release-note
Fix(image): get convert and extract image from env
```

### CODE REVIEW

File: testutil/testutil.go [hunk 1]
```diff
@@ -2,21 +2,17 @@ package testutil
...
-	presetImageConfigMap := &corev1.ConfigMap{}
-	presetImageConfigMap.Data = map[string]string{
-		"savedmodel-extract":       "demo.goharbor.com/release/savedmodel:v0.2",
-		"ormb-storage-initializer": "demo.goharbor.com/release/savedmodel:v0.2",
-	}
-
-	return presetImageConfigMap
+func InitPresetModelImage() {
+	os.Setenv("SAVEDMODEL_EXTRACT_IMAGE", "demo.goharbor.com/release/savedmodel:v0.2")
+	os.Setenv("ORMB_INITIALIZER_IMAGE", "demo.goharbor.com/release/savedmodel:v0.2")
```
Comment (FogDong) [inline testutil/testutil.go:15]: ping /cc @simon-cj I'm not sure about the image in here.

**Comment by simon-cj** [comment]
CI is not passed

File: testutil/testutil.go (hunk 1 above)
Comment (simon-cj) [reply testutil/testutil.go:15]: Tt will check string format, not check the content.
Maybe there need all of the env key for all image.

File: pkg/controllers/consts.go [hunk 2]
```diff
@@ -30,3 +26,19 @@ var (
...
+	"graphdef-extract":    "GRAPHDEF_EXTRACT_IMAGE",
+	"h5-extract":          "H5_EXTRACT_IMAGE",
+	"mxnetparams-extract": "MXNETPARAMS_EXTRACT_IMAGE",
+	"onnx-extract":        "ONNX_EXTRACT_IMAGE",
+	"savedmodel-extract":  "SAVEDMODEL_EXTRACT_IMAGE",
+	"torchscript-extract": "TORCHSCRIPT_EXTRACT_IMAGE",
+	"pmml-extract":        "PMML_EXTRACT_IMAGE",
+	"caffemodel-convert":       "CAFFE_CONVERSION_IMAGE",
+	"mxnetparams-convert": "MXNET_CONVERSION_IMAGE",
+	"h5-convert":          "H5_CONVERSION_IMAGE",
```
Comment (simon-cj) [inline pkg/controllers/consts.go:42]: Add the netdef-convert, it is newest the conversion.

**Pull request merged**